internal/pkg/cli/dialog/workspaces.go

```diff
@@ -16,7 +16,7 @@ func (p *Dialogs) AskWorkspace(
 	if d.IsSet(`workspace-id`) {
 		workspaceID := d.GetString(`workspace-id`)
 		for _, w := range allWorkspaces {
-			if string(w.Sandbox.ID) == workspaceID {
+			if string(w.Config.ID) == workspaceID {
```

Comment:
Proč se to tady mění?

Reply:
Jo, to jsem zapomněl popsat. `sandboxesapi.Get` potřebuje config ID, ne instance ID. Takže se to mění, aby "user-facing" ID bylo config ID.